snap/snapcraft.yaml

```diff
@@ -194,10 +196,27 @@ apps:
     daemon: oneshot
     start-timeout: 15m
     plugs: [network]
+  # This is a simple service which calls into vault to retrieve the redis password
+  # and then makes a redis call to set the password. As such, redis is now started after
+  # security-secretstore-setup has run, following by this service. Once the password has
+  # been set, this process simply sleeps forever (in theory due to issues with one-shot
+  # services in the docker world).
```

Comment:
No longer sleeps forever. Update comment for actual behavior.

Reply:
Done